### ISSUE TITLE
Docker instance runs but no web server

### ISSUE THREAD

**Issue by Vap0r1ze** [krateng/maloja#94]
Built from [`Dockerfile`](https://github.com/krateng/maloja/blob/a988f0e6dc700ff4924b4069764ae5420ed7bc42/Dockerfile)

Steps to reproduce:
```
git clone https://github.com/krateng/maloja
cd maloja
docker build -t majola .
docker run majola
```

[Build Logs](https://gist.github.com/Vap0r1ze/29037753973625eac731c5da1f33ceed)

Instance Logs:
```
Exception in thread Thread-10 (self_scheduling_func):
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/threading.py", line 1009, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.10/threading.py", line 946, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.10/site-packages/doreah/regular.py", line 149, in self_scheduling_func
    now = datetime.datetime.now(tz=tz())
  File "/usr/local/lib/python3.10/site-packages/doreah/regular.py", line 17, in tz
    return datetime.timezone(offset=datetime.timedelta(hours=config["offset"]))
TypeError: unsupported type for timedelta hours component: NoneType
Exception in thread Thread-12 (self_scheduling_func):
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/threading.py", line 1009, in _bootstrap_inner
Exception in thread Thread-13 (self_scheduling_func):
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/threading.py", line 1009, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.10/threading.py", line 946, in run
    self.run()
  File "/usr/local/lib/python3.10/threading.py", line 946, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.10/site-packages/doreah/regular.py", line 71, in self_scheduling_func
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.10/site-packages/doreah/regular.py", line 149, in self_scheduling_func
    now = datetime.datetime.now(tz=tz())
  File "/usr/local/lib/python3.10/site-packages/doreah/regular.py", line 17, in tz
    return datetime.timezone(offset=datetime.timedelta(hours=config["offset"]))
TypeError: unsupported type for timedelta hours component: NoneType
    now = datetime.datetime.now(tz=tz())
  File "/usr/local/lib/python3.10/site-packages/doreah/regular.py", line 17, in tz
    return datetime.timezone(offset=datetime.timedelta(hours=config["offset"]))
TypeError: unsupported type for timedelta hours component: NoneType
Exception in thread Thread-14 (self_scheduling_func):
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/threading.py", line 1009, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.10/threading.py", line 946, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.10/site-packages/doreah/regular.py", line 149, in self_scheduling_func
    now = datetime.datetime.now(tz=tz())
  File "/usr/local/lib/python3.10/site-packages/doreah/regular.py", line 17, in tz
    return datetime.timezone(offset=datetime.timedelta(hours=config["offset"]))
TypeError: unsupported type for timedelta hours component: NoneType
```

**Comment by Vap0r1ze** [comment]
Also just realized I misspelled the project name in the docker commands lol

**Comment by krateng** [comment]
Could you inspect your container? Are `default.ini` and `settings.ini` present in the settings folder (`/etc/maloja/settings` or `~/.local/share/maloja/settings`)?

**Comment by FoxxMD** [comment]
In a fresh container **without** `MALOJA_SKIP_SETUP=yes`

```
/usr/src/app # ls /etc/maloja/settings
default.ini
```

with skip setup `settings.ini` is generated.

`default.ini` does contain `TIMEZONE = 0`

**Comment by Vap0r1ze** [comment]
My bad, I read the Dockerfile and thought that EXPOSE actually published the port (just read the docs on it). Publishing the port didn't solve it though, but setting that environment variable did (`MALOJA_HOST=0.0.0.0`). That should definitely be default IMO but I see that this commit was only a couple hours ago

**Comment by krateng** [comment]
So the webserver is accessible, but the TypeError above still happens?

**Comment by Vap0r1ze** [comment]
> So the webserver is accessible, but the TypeError above still happens?

Yes it does

**Comment by krateng** [comment]
Is this in a container that already ran a previous version? Does your `settings.ini` file contain anything?

**Comment by krateng** [comment]
I think I might have figured out the issue, I added a temporary fix to 2.12.20, let me know if that fixes it.

**Comment by FoxxMD** [comment]
The fix works for me on a fresh build.
